### PR TITLE
Call use-local-coherence as part of CI-test

### PR DESCRIPTION
**Pull request by pranavkm**
cc @cesarbs

### CODE REVIEW

File: makefile.shade [hunk 1]
```diff
@@ -296,13 +296,19 @@ var buildTarget = "compile"
...
     @{
         if (string.IsNullOrEmpty(coherencePath))
         {
             throw new Exception("COHERENCE_PATH not specified.");
         }
 
+        universeCommitsFile = Path.Combine(coherencePath, UniverseCommitsFileName);
+        if (!File.Exists(universeCommitsFile))
+        {
+            throw new Exception("universeCommitsFile could not be found at " + UNIVERSE_COMMITS_FILE);
```
Comment (cesarbs) [inline makefile.shade]: `+ universeCommitsFile`

Change the beginning of the sentence to "Universe commits file".

File: makefile.shade [hunk 2]
```diff
@@ -296,13 +296,19 @@ var buildTarget = "compile"
...
 
-#cache-coherence
+#use-local-coherence description='Sets up environment variables to use Coherence at COHERENCE_PATH'
     @{
         if (string.IsNullOrEmpty(coherencePath))
         {
             throw new Exception("COHERENCE_PATH not specified.");
         }
 
+        universeCommitsFile = Path.Combine(coherencePath, UniverseCommitsFileName);
```
Comment (cesarbs) [inline makefile.shade]: This overrides the first value set if the the `UNIVERSE_COMMITS_FILE` environment variable was set. Check if it's null/empty before doing this.

Comment (cesarbs) [inline makefile.shade]: Also make sure you unset `UNIVERSE_COMMITS_FILE` on the CI configs since we want this code to build the path to it.

Comment (pranavkm) [inline makefile.shade]: :+1:

**Comment by pranavkm** [comment]
Updated

**Comment by cesarbs** [comment]
:ship:

**Pull request closed without merging**